scipy/linalg/matfuncs.py

```diff
@@ -668,3 +669,60 @@ def rounded_sign(x):
         return S0
     else:
         return S0, errest
+
+
+def khatri_rao(a, b):
+    r"""
+    Khatri-rao product
+    A column-wise Kronecker product of two matrices
+
+    Parameters
+    ----------
+    a:  (n, k) arraylike
+        Input array
+    b:  (m, k) arraylike
+        Input array
+
+    Returns
+    -------
+    c:  (n*m, k) ndarray
+        Khatri-rao product of `a` and `b`.
+
+    Notes
+    -----
+    The mathematical definition of the Khatri-Rao product is:
+    .. math::
```

Comment:
This doesn't render correctly, needs a blank line above `.. math::`. Please look at the rendered docs page in the CI job "doc build artifact"